webcomix/supported_comics.py

```diff
@@ -151,4 +151,112 @@
+    },
+    "StationV3": {
+        "name": "StationV3",
+        "start_url": "https://www.stationv3.com/d3/20170101.html",
+        "comic_image_selector": "((//div[@id='middle']//center)[1]//img/@src)[5]",
+        "next_page_selector": "(//div[@id='middle']//center)[2]//a[img[contains(@src, 'http://www.stationv3.com/images/next2.gif')]]/@href",
+    },
+    "SabrinaOnline": {
+        "name": "SabrinaOnline",
+        "start_url": "https://www.sabrina-online.com/1996-01.html",
+        "comic_image_selector": "//h3//img[contains(@src, 'SO')]/@src",
+        "next_page_selector": "(//a[img[contains(@alt, 'NEXT')]]/@href)[last()]",
+    },
+    "TheSwordInterval": {
+        "name": "TheSwordInterval",
+        "start_url": "https://swordinterval.thecomicseries.com/comics/1/#content-start",
+        "comic_image_selector": "//img[@id='comicimage']/@src",
+        "next_page_selector": "//a[contains(@rel, 'next')]//@href",
+    },
+    "QuestionableContent": {
+        "name": "QuestionableContent",
+        "start_url": "https://questionablecontent.net/view.php?comic=1",
+        "comic_image_selector": "//img[@id='strip']/@src",
+        "next_page_selector": "//a[contains(text(), 'Next')]//@href",
+    },
+    "VGCats": {
+        "name": "VGCats",
+        "start_url": "https://www.vgcats.com/comics/?strip_id=0",
+        "comic_image_selector": "(//table//img[contains(@src, 'images')]/@src)[1]",
+        "next_page_selector": "//table//a[img[contains(@src, 'next')]]/@href",
+    },
+    "SuperEffective": {
+        "name": "SuperEffective",
+        "start_url": "https://vgcats.com/super/index.php?strip_id=0",
+        "comic_image_selector": "(//table//img[contains(@src, 'images')]/@src)[1]",
+        "next_page_selector": "//table//a[img[contains(@src, 'next')]]/@href",
+    },
+    "RivalQuickies": {
+        "name": "RivalQuickies",
+        "start_url": "https://vgcats.com/rivals/?strip_id=0",
+        "comic_image_selector": "(//table//img[contains(@src, 'images')]/@src)[1]",
+        "next_page_selector": "//table//a[img[contains(@src, 'next')]]/@href",
+    },
+    "VisionHaze": {
+        "name": "VisionHaze",
+        "start_url": "https://www.visionhaze.com/index.php?p=1",
+        "comic_image_selector": "//div[@class='page']//img/@src",
+        "next_page_selector": "//div[@class='navbutton']//a[contains(text(), '›')]/@href",
+    },
+    "TheJunkHyenasDiner": {
+        "name": "TheJunkHyenasDiner",
+        "start_url": "https://junkhyenasdiner.com/comic/intro/",
+        "comic_image_selector": "//div[@id='comic']//img/@src",
+        "next_page_selector": "//a[@class='navi comic-nav-next navi-next']/@href",
+    },
+    "JackRabbit": {
+        "name": "JackRabbit",
+        "start_url": "https://jackrabbit.thecomicseries.com/comics/1/#content-start",
+        "comic_image_selector": "//img[@id='comicimage']/@src",
+        "next_page_selector": "//a[contains(@rel, 'Next')]//@href",
```

Comment:
```suggestion
        "next_page_selector": "//a[contains(@rel, 'next')]//@href",
```

At first glance, after running the tests and validating it manually, this change to the XPath seems to do the trick

Reply:
Strange, when I run it the capitol letter works and the web page uses a 'N' vs an 'n'
I don't think scrapy should care about case sensitivity here so it should be fine

Reply:
ignore the re-request for a moment I'm still commiting it